docs/getting-started.md

```diff
@@ -0,0 +1,104 @@
+
+## Step 2. Configure your store
+
+Virto Commerce platform is 4-MULTI: multi-language, multi-currency, multi-theme and multi-store by design. This gives the ability to have and run multiple stores on the same system.
+
+To configure your store you must accomplish a set of actions:
+
+1. Create first store (enter store code and name)
+2. Select your catalog
+3. Select available currency, languages
+4. Configure advanced properties:
+    1. Store description, emails and URLs
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
+
+Open your first store in a browser.
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
+
+In the Virto Commerce, a pricelist is a set of pricing details that can be assigned to items. All prices are stored in a pricelist(s), which have specific currency set. When a pricelist is created, you define which products will be included into this pricelist and what prices will be defined for them.
+
+The way to apply the prices to products on Storefront is creating a Pricelist Assignment. It binds a catalog with pricelist and can add optional conditions for this binding.
+
+Pricelist assignment identifies the pricelist from which a product price is shown for customers. Moreover, pricelist assignments can do much more than simply provide default list and sale prices for products. This feature allows you to provide custom prices to targeted customers according to customer's characteristics such as age, gender, geographic location and searched terms.
+
+Thus, price configuring consists of the following actions:
+
+1. Create pricelist
+2. Add items to the pricelist
+
+3. Create pricelist assignment
+4. Manage inventory
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipment methods. Virto Commerce provides the ready to use module which you can set your own configurations:
+
+* Install Payment module (For example: PayPal)
+* Configure [Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md) method
```

Comment:
Just not sure, if ShipStation integration module is in scope of "Getting started". @OlegoO  ?
Maybe, link to https://github.com/VirtoCommerce/vc-module-shipping is enough?

Reply:
unfortunately, there are no descriptions in https://github.com/VirtoCommerce/vc-module-shipping